app/assets/stylesheets/admin/_overrides.scss

```diff
@@ -25,3 +25,11 @@ code {
     display: block;
   }
 }
+
+.app-c-translated-textarea, .app-c-translated-input {
+  & > *:first-child {
+    .govuk-form-group {
+      margin-bottom: 10px !important; // stylelint-disable-line declaration-no-important
+    }
+  }
+}
```

Comment:
A few small things on this now that I've seen this in context (sorry I didn't think of this earlier):
- I've realised there is a built-in way to override the design-system margins, which is to use the [Spacing helpers](https://design-system.service.gov.uk/styles/spacing/#applying-spacing-in-your-own-css), which means we don't need the `!important` declaration (which feels better :smile:). 
- It's best not to use explicit pixel dimensions but rather use the [Responsive spacing scale](https://design-system.service.gov.uk/styles/spacing/#responsive-spacing) so that the spacing can adapt for different viewports. 
- we shouldn't need the `& > *:first-child` rule to target the `govuk-form-group` class (provided that it's the only use of that class within the parent element which I think is the case here).

So I think this can be written as
```
.app-c-translated-textarea, .app-c-translated-input {
  .govuk-form-group {
    @include govuk-responsive-margin(2, "bottom");
  }
}
```

Reply:
Thanks @davidtrussler ! Unfortunately I can't seem to remove the need for !important... maybe would be good to pair on this tomorrow, but I've incorporated the other changes here.